jbi/services/jira.py

```diff
@@ -30,12 +30,23 @@
 
 JIRA_DESCRIPTION_CHAR_LIMIT = 32767
 
+
+def fatal_code(exc):
+    """Do not retry 4XX errors, mark them as fatal."""
+    try:
+        return 400 <= exc.response.status_code < 500
```

Comment:
Do we want to be more granular than this? For example, we'd want to retry on a `429 Too Many Requests` right? That's why we're using the `backoff.expo` strategy, after all.